### ISSUE TITLE
[Bug]: Getting router is undefined when the page reloads after user logged

### ISSUE THREAD

**Issue by phpmathan** [soybeanjs/soybean-admin#436]
### Software Version

v1.1.0

### Operating Environment

Ubuntu

### Operating Architecture

AMD64

### Reproduce Steps

After the user logged in, if the page reloads and uses static auth routes, then we get router is undefined error the page is not rendering because of no auth routes.

### What is the expected result?

Replace the following import in src/store/modules/tab/index.ts

import { useRouter } from 'vue-router'; to import { router } from '@/router';

### What is the actual result?

After replacing the router from 'vue-router' to '@/router' it is working fine, hence the behavior is same.

**Comment by honghuangdc** [comment]
could you give me the more detail reproduction

**Comment by phpmathan** [comment]
I've created the modular structure for easy code management and reuse the modules, and I initialize the modules before the setupRouter in main.ts, during the module initialization I prepare the static routes which internally call tabStore.initHomeTab() which triggered the error router is undefined.

![20240514200107](https://github.com/soybeanjs/soybean-admin/assets/527140/b64ebba5-b0c7-4c99-9e34-53bed85e8f4e)

![20240514200255](https://github.com/soybeanjs/soybean-admin/assets/527140/95aa8839-fc45-4725-91ee-9082a14391d7)

Changing import { useRouter } from 'vue-router'; to import { router } from '@/router'; in src/store/modules/tab/index.ts solves my problem. I created this ticket for users like me who customize may see such issues, and these notes may help them.
